lib/trino-record-decoder/src/test/java/io/trino/decoder/DecoderTestColumnHandle.java

```diff
@@ -70,6 +72,22 @@
      */
     private final boolean internal;
 
+    private final Optional<List<String>> oneOfs;
+
+    public DecoderTestColumnHandle(
```

Comment:
all ctors should ideally go via the `JsonCreator`, less chance of mistakes

Reply:
if this causes a large diff you can extract as the first commit with some reasons in commit message